api/controllers/MobileAppController.js

```diff
@@ -1,8 +1,22 @@
 module.exports = {
-
   updateInfo: function (req, res) {
-    var result = {}
```

Comment:
Since the new implementation is entirely in the other method, you can remove this method and the routes.js entry associated with it completely. The old apps are no longer accessing it because they just get redirected to hylo.com/newapp.